File: pe10.py
def prime(n):
	f=0
	for i in range(2, (n//2)+1):
		if(n%i==0):
			return False
	return True

s=2
for n in range(3, 2000000, 2):
	if(prime(n)):
		s+=n
	print(n, s)
print(s)

File: pe10.java
class pe10
{
	static boolean prime(long n)
	{
		for(long i=2;i<n;i++)
			if(n%i==0)
				return false;
		return true;
	}

	public static void main(String args[])
	{
		long i,s=2;
		for(i=3;i<2000000;i+=2)
			if(prime(i)==true)
				s+=i;
		System.out.print(s);
	}
}